test/Microsoft.IdentityModel.TestUtils/IdentityComparer.cs

```diff
@@ -673,6 +673,74 @@ public static bool AreAudienceValidationResultsEqual(object object1, object obje
             return context.Merge(localContext);
         }
 
+        public static bool AreLifetimeValidationResultsEqual(object object1, object object2, CompareContext context)
+        {
+            var localContext = new CompareContext(context);
+            if (!ContinueCheckingEquality(object1, object2, context))
+                return context.Merge(localContext);
+
+            return AreLifetimeValidationResultsEqual(
+                object1 as LifetimeValidationResult,
+                object2 as LifetimeValidationResult,
+                "LifetimeValidationResult1",
+                "LifetimeValidationResult2",
+                null,
+                context);
+        }
+        internal static bool AreLifetimeValidationResultsEqual(
```

Comment:
nit: should have a new line between methods

Reply:
Fixed.